Lombiq.Hosting.Tenants.EmailQuotaManagement/Services/IQuotaService.cs

```diff
@@ -34,7 +34,17 @@ public interface IQuotaService
     void ResetQuota(EmailQuota emailQuota);
 
     /// <summary>
-    /// Saves the given quota.
+    /// Saves the given quota on reminder sent.
     /// </summary>
-    void SaveQuota(EmailQuota emailQuota);
+    void SaveQuotaReminder(EmailQuota emailQuota);
+
+    /// <summary>
+    /// Returns <see langword="true"/> if the reminder email should be sent.
```

Comment:
None of the comments here are much more than the method name as a sentence. These should explain more than what's trivial.